pyciemss/ouu/qoi.py

```diff
@@ -0,0 +1,22 @@
+import numpy as np
+
+
+def scenario2dec_nday_average(
```

Comment:
Can we remove this function, or make it model/scenario agnostic?

Reply:
I have made it more general and kept it for now to be used in tests and stuff. 
In the notebook I copy-pasted the function to make it explicit that the qoi function is user-defined callable function. Do you think that is better or just importing it as an example of qoi function (less code in the notebook) is better?

Reply:
That's good. Thanks!